tokio-trace-env-logger/examples/hyper-echo.rs

```diff
@@ -18,17 +18,17 @@ use std::str;
 mod sloggish;
 use self::sloggish::SloggishSubscriber;
 
-use tokio_trace::Level;
+use tokio_trace::{field::Value, Level};
 use tokio_trace_futures::{Instrument, Instrumented};
 
 type BoxFut = Box<dyn Future<Item = Response<Body>, Error = hyper::Error> + Send>;
 
 fn echo(req: Request<Body>) -> Instrumented<BoxFut> {
     span!(
         "request",
-        method = req.method(),
-        uri = req.uri(),
-        headers = req.headers()
+        method = &Value::debug(req.method()),
+        uri = &Value::debug(req.uri()),
+        headers = &Value::debug(req.headers())
```

Comment:
To avoid the `&Value::debug` call, would this require a Value impl in the `http` crate, or does this need to be done in core? I'm trying to think of an a workaround that doesn't run afoul of orphan rules.

Reply:
I have some thoughts on this that I'm happy to show you.

Reply:
I'd love to see!